api/src/main/java/org/openmrs/api/db/hibernate/HibernatePersonDAO.java

```diff
@@ -170,34 +115,29 @@ public void setSessionFactory(SessionFactory sessionFactory) {
 		names.getList().forEach(x -> people.add(getPerson((Integer) x[0])));
 		
 		return people;
-	}	
+	}
+	
 	
 	/**
-	 * This method executes a search on either firstName or lastName attributes of person
-	 * 
-	 * @param firstName true if search should be executed on firstname only
-	 * @param name to be searched in soundex representation
-	 * @param birthyear to be matched
-	 * @param includeVoided true if voided persons should be matched
+	 * This method executes a Lucene search on persons based on the soundex filter with two search names given
+	 *
+	 * @param name1 firstName to be searched
+	 * @param name2 last name to be searched 
+	 * @param birthyear the birthyear the searched person should have 
+	 * @param includeVoided true if voided person should be included 
 	 * @param gender of the person to search for 
-	 * @return the persons that match the search criteria as defined
+	 * @return the set of Persons that match the search criteria
 	 */
-	private Set<Person> execSoundexLuceneQueryByNames(boolean firstName, String name, Integer birthyear, boolean includeVoided , String gender) {
+	private Set<Person> executeSoundexTwoPersonNamesQuery(String name1, String name2, Integer birthyear, boolean includeVoided , String gender) {
 		PersonLuceneQuery personLuceneQuery = new PersonLuceneQuery(sessionFactory);
```

Comment:
I think using the firstName and lastName instead of name1 and name2 would give the developers an apparent understanding of what these variables are. But I see you have put the definitions of those params in java comments. Still, I suggest changing them to firstName and lastName for the sake of simplicity. 🙂

Reply:
@gravewalker666 Thanks for your feedback. I applied the changes accodingly

Reply:
Fixed